src/social-network-provider/twitter.com/ui/group.ts

```diff
@@ -0,0 +1,19 @@
+import { SocialNetworkUI } from '../../../social-network/ui'
+import { MessageCenter } from '../../../utils/messages'
+import Services from '../../../extension/service'
+import { PersonIdentifier } from '../../../database/type'
+
+export const PreDefinedTwitterGroupNames = {
```

Comment:
Shouldn't it share with global?

Reply:
Any better place to define it?

Reply:
Maybe `PreDefinedVirtualGroupNames` in src\database\type.ts